addons/medical_treatment/ACE_Medical_Treatment.hpp

```diff
@@ -570,7 +570,9 @@ class ADDON {
         // The number of doses over maxDose where there is a chance to overdose.
         // Example with maxDose = 4 and maxDoseDeviation = 2: Dose 4: Safe | Dose 5 and 6: Possible overdose | Dose 7: Guaranteed overdose
         maxDoseDeviation = 2;
-        // Function to execute upon overdose. Arguments passed to call back are 0: unit <OBJECT>, 1: medicationClassName <STRING>
+        // The dose of the medication, to allow for multiple dose types of the same medication
```

Comment:
Comment doesn't resonate with me as it is, `multiple dose types` doesn't make sense to me.

I'd suggest the following, but I'm hoping someone else can come up with a better explanation:
```suggestion
        // The dose of the medication, to allow for different dose amounts of the same medication
```